### PR TITLE
chore: Remove legacy Google analytics telemetry

### LINKED ISSUE #6620: Remove legacy telemetry events
We've had full coverage for our new unstructured telemetry events for some time now, presumably all major bugs have been spotting, and the downstream Squared dbt transformations [don't rely on the legacy events](https://github.com/meltano/squared/issues/340) as a primary source. We can go ahead and remove any extra code for these legacy events and stop double sending a backup structured event (if were still doing that).

cc @WillDaSilva @tayloramurphy

### PR DESCRIPTION
**Pull request by WillDaSilva**
Closes #6620

### CODE REVIEW

File: src/meltano/cli/install.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from meltano.core.project_plugins_service import ProjectPluginsService
 from meltano.core.tracking import CliEvent, PluginsTrackingContext
 
+from ..core.tracking.tracker import Tracker
```
Comment (pandemicsyn) [inline src/meltano/cli/install.py:11]: I know this technically works, just making sure it's intentional (ditto for the similar one below).

**Review by pandemicsyn — APPROVED** [review]
🥳 Screen Shot 2022-08-15 at 7 04 09 PM 🎉 

One question inline but generally lgtm - and super glad to have this gone!

File: src/meltano/cli/install.py (hunk 1 above)
Comment (WillDaSilva) [reply src/meltano/cli/install.py:11]: Good catch. It must've been added automatically, since I certainly didn't write that. I'll remove it.

Comment (pandemicsyn) [reply src/meltano/cli/install.py]: The auto import hook from my editor used to do this occasionally - so if you come across some pre-existing ones, they're probably my fault 😁

**Pull request merged**